spec/models/piece_spec.rb

```diff
@@ -8,4 +8,36 @@
         .to eq('white-queen.png')
     end
   end
+  describe "#move_to!" do
+    let(:king) { create(:king, x_position: 1, y_position: 1) }
```

Comment:
It worked really well.  The downside is I didn't use factories to create my pieces.